azext_iot/central/params.py

```diff
@@ -386,3 +384,80 @@ def load_central_arguments(self, _):
             options_list=["--description", "--desc"],
             help="Detailed description of the job.",
         )
+
+    with self.argument_context("iot central query") as context:
+        context.argument(
+            "query_string",
+            options_list=["--query-string", "--qs"],
+            help="Query clause to retrieve telemetry or property data.",
+        )
+        context.argument(
+            "api_version",
+            options_list=["--api-version", "--av"],
+            choices=CaseInsensitiveList([ApiVersion.v1_1_preview.value]),
+            default=ApiVersion.v1_1_preview.value,
+            help="The API version for the requested operation.",
+        )
+
+    with self.argument_context("iot central export destination") as context:
+        context.argument(
+            "destination_id",
+            options_list=["--dest-id"],
+            help="Unique identifier for the destination.",
+        )
+        context.argument(
+            "api_version",
+            options_list=["--api-version", "--av"],
```

Comment:
You should be able to delete this `--api-version` arg definition as `iot central export destination` should inherit `--api-version` from `iot central export`.

Reply:
Removed